azure/app-configuration-dynamic-reload/Models/PageSettings.cs

```diff
@@ -0,0 +1,9 @@
+namespace app_configuration_dynamic_reload.Models;
+
+public class PageSettings
```

Comment:
Depending on the use, I would consider making this a `record`. That would create value based equality, and add a ToString, copy constructor, and with expression support.